guides/source/initialization.md

```diff
@@ -32,11 +32,11 @@ Launch!
 -------
 
 Let's start to boot and initialize the app. A Rails application is usually
-started by running `rails console` or `rails server`.
+started by running `bin/rails console` or `bin/rails server`.
 
 ### `railties/exe/rails`
 
-The `rails` in the command `rails server` is a ruby executable in your load
+The `rails` in the command `bin/rails server` is a ruby executable in your load
 path. This executable contains the following lines:
```

Comment:
This is no longer accurate; the `rails` binstub isn't (necessarily) on the user's load path.

This section and the one following it describe how the global `rails` command bootstraps the application's binstub, which is the functionality we're trying to deemphasise with these updates. I think you can just delete them and move `bin/rails` up to be the first section.

Reply:
@eugeneius done 👍🏼